src/package.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
@ptewson-sfdo - Ran 'cci task run update_package_xml' after removing the custom label. Does this look normal?

Reply:
Oh - good catch. I think we are safe to disregard this and it sounds like removing that attribute is in line with expected behavior for the cci task.